cg_hermes/config/tags.py

```diff
@@ -119,6 +119,8 @@
     "chanjo": {"description": "Tool to keep track of coverage over specific regions"},
     "chromograph": {"description": "Tool to create PNG images from BED and WIG files from mikaell"},
     "cnvkit": {"description": "Tool to call copy number variations"},
+    "ascat-ngs": {"description": "Tool to identify somatically acquired copy-number alterations"},
+    "delly": {"description": "Structural variant prediction tool"},
```

Comment:
```suggestion
    "delly": {"description": "Cancer structural variant prediction tool "},
```

can you also remove `mutect` and `strelka` from below lines. As these callers no more exists in BALSAMIC